.github/workflows/first_time_contributor.yml

```diff
@@ -0,0 +1,30 @@
+name: First-time Contributor Welcome
+
+on:
+    issues:
+      types: [opened]
+    pull_request_target:
+      types: [opened, closed]
+
+jobs:
+  first-time-contributor-welcome:
+    runs-on: ubuntu-latest
+    steps:
+    # We don't use the actions/first-interaction action because it can't reference the author, nor can it comment after
+    # PR merge.
+    # Once https://github.com/wow-actions/welcome/pull/12 is merged, we should use https://github.com/wow-actions/welcome
+    # directly.
+    - uses: OrchardCMS/welcome-action@task/update-to-node-20
+      with:
+        FIRST_ISSUE_COMMENT: >
+          Thank you for submitting your first issue, awesome! 🚀 If you haven't filled out the template fully already,
+          please be sure to do so. This helps us better understand your feature request/bug report. A core team member 
+          will be with you shortly to review your issue.
+        FIRST_PR_COMMENT: >
+          Thank you for submitting your first pull request, awesome! 🚀 If you haven't read it already, please be sure 
+          to check out our [contribution guide](https://docs.orchardcore.net/en/latest/docs/guides/contributing/). A 
+          core team member will be with you shortly to review your PR.
+        FIRST_PR_MERGED_COMMENT: >
+          Congratulations on your first PR merge! 🎉 Thank you for your contribution! We're looking forward to welcome 
+          other contributions of yours in the future. @all-contributors please add @{{ author }} for code.
+        STAR_MESSAGE: If you like Orchard Core, please star our repo!
```

Comment:
```suggestion
        STAR_MESSAGE: If you like Orchard Core, please star our repo and join our [community channels ](https://github.com/OrchardCMS/OrchardCore/?tab=readme-ov-file#get-in-touch)
```

Reply:
How about linking rather to https://github.com/OrchardCMS/OrchardCore?tab=readme-ov-file#help-and-support, or both?

Reply:
I think social media is better so they can stay engaged with us.